contents/blog/interview-snapshot-guide.md

```diff
@@ -0,0 +1,82 @@
+---
+date: 2023-03-06
+title: "Interview Snapshots: Our 80/20 approach to sharing user insights"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - annika-schmid
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Inside PostHog
+tags:
+  - Guides
+  - Product engineers
+---
+
+Regularly talking to users is an important habit for any product team, but it's wasted without an efficient way to share what you learn. At PostHog, we favor interview snapshots. Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.co.uk/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), we like snapshots because:
+
+ 1. They're easy to create – 30 minutes maximum if you do them right after an interview.
+ 2. They're easy to digest – no scrolling necessary, just drop a screenshot into Slack.
+ 3. They're conversation starters – each individual snapshot triggers lively debate and ideas.
+
+In this guide, I'll share how we create user interview snapshots at PostHog.
+
+## Interview snapshots explained
+
+Snapshots are an easy-to-digest summary of the insights you glean from interviews. Ideally you take those notes during the interview, or you can ask one of your colleagues to do so – sometimes it's easier if one person talks and the other takes notes. 
+
+Here's what a completed snapshot looks like:
+
+![interview-snapshot](../images/interview-snapshots/interview-snapshot-filled-out.jpg)
+
+While taking notes, focus on how the user is using your product, and how they feel while using it. What makes them excited? What frustrates them? Those are the insights that later go into your interview snapshot.
+
+Another approach is to record the conversation then transcribe and summarise it later. This might seem easier, but I don't recommend it because it's more time consuming – remember we're optimizing for efficiency here. You can, of course, still record interviews as a useful reference for your notes, or for extracting direct quotes later.
+
+Instead, I recommend blocking 30 minutes after your interview to review your notes and fill out the user interview snapshot while it's fresh in your mind. This way, each interview becomes a simple 60-minute block – 30 minutes for your interview, and 30 minutes for review.
+
+## Creating your interview snapshot
+
+Once you've reviewed your notes, you're ready to start filling out your snapshot. You can use the template below, or recreate it in Google Docs, Figma etc.
```

Comment:
```suggestion
Once you've reviewed your notes, you're ready to start filling out your snapshot. You can get a copy from the Interview Snapshot template from [Figma](https://www.figma.com/file/iO6kIZ8kXdMudLtu7kSyc5/PostHog-Interview-Snapshot-Template?node-id=0%3A1&t=LDU8ALaROQyfh6JL-1) or [Google Drive](https://drive.google.com/file/d/1Md-yLMtgGTEfjUzSFgUur5I-5H8lSYZp/view).
```

Reply:
Sorry for Google Drive I meant Google Slides (ie. sharing an editable template is what would be useful). If it's just in Figma, that's fine, we can ignore the PDF version!

Reply:
I'm making a Slides version. Won't take long.